app/views/spree/shared/_order_details.html.haml

```diff
@@ -13,9 +13,9 @@
     .pad
       .text-big
         = t :order_payment
-        %strong=  OrderPaymentFinder.last_payment_method(order)&.name
+        %strong=  last_payment_method(@order)&.name
```

Comment:
You substituted `order` with `@order`. Was that intentional. I think it's better to use `order` because it makes clear where the variable comes from and makes it easier to re-use partials (compared to ivars).

Reply:
Okay, I looked it up. This partial is only used in `app/views/spree/orders/show.html.haml` which also has `@order` defined. So this change doesn't break anything but the partial contains a dozen references to `order` and none to `@order`. So I think it would be a lot clearer to use `order` here. I will change it.